sdk/client.go

```diff
@@ -112,6 +112,35 @@ func (cli *Client) GetSubtopicsPublic(ctx context.Context, reqHeaders Headers, i
 	return &subtopics, nil
 }
 
+// GetNavigationPublic gets the public list of navigation items
+func (cli *Client) GetNavigationPublic(ctx context.Context, reqHeaders Headers, lang string) (*models.Navigation, error) {
```

Comment:
So instead of passing in lang parameter in signature, it would be better to create an Options type that contains a list of possible query parameters, that way the method signature will not have to change as new query parameters are added.

An example of this is: https://github.com/ONSdigital/dp-search-reindex-api/blob/develop/sdk/v1/client.go#L237

Reply:
This has been done